search-app/docs/api.md

```diff
@@ -4670,7 +4670,7 @@ __Sample response__
 ```
 #### <a name="subscription-access-control"></a> Subscription Access Control
 
-Access to subscription is granted through the provider via the INGEST_MANAGEMENT_ACL and EMAIL_SUBSCRIPTION_MANAGEMENT
+Access to subscription is granted through the provider via the INGEST_MANAGEMENT_ACL and SUBSCRIPTION_MANAGEMENT
```

Comment:
So a user needs to have INGEST_MANAGEMENT_ACL in addition to SUBSCRIPTION_MANAGEMENT to CRUD Subscriptions?

Reply:
Yes. INGEST_MANAGEMENT ACL was there before.

Reply:
I mean we want to control who can ingest with INGEST_MANAGEMENT ACL for all concepts.

Reply:
I thought we could have Subscriptions just be managed by SUBSCRIPTION_MANAGEMENT. But if it needs the other one, I think that is ok, we just need to be very clear about it so everyone knows. That it is here in the documentation is helpful.

Reply:
I couldn't tell exactly what kind of permissions are needed to update/delete a subscription concept as I can't find API doc or tests describing the expected behavior.

Reply:
As far as I can tell, update/delete subscription requires "update" permission granted through both INGEST_MANAGEMENT_ACL and SUBSCRIPTION_MANAGEMENT ACL.  So, I moved this part to the Ingest api doc. Not sure why INGEST_MANAGEMENT_ACL for other concepts are in the search api doc, but absent in the ingest doc.   For the search api doc here, I don't see INGEST_MANAGEMENT_ACL plays a role, so I just added the SUBSCRIPTION_MANAGEMENT here.

Reply:
The permissions should be listed in Ingest API doc. The reason Ingest Management ACL is mentioned in search API doc is because we don't ingest of certain concepts in the search API, e.g. tags and we need to mention it for that. We should constantly improve the API docs.